client/src/pages/admin/AdminLibraryForm.tsx

```diff
@@ -170,6 +177,22 @@ function AdminLibraryForm() {
                   register={register}
                   label="Czas rezerwacji"
                 />
+                <TextInputField
+                  errors={errors}
+                  field="emailAddress"
+                  register={register}
+                  label="Adres e-mail"
+                />
+                <MuiTelInput
+                  defaultCountry="PL"
+                  continents={['EU']}
+                  value={phoneNumberValue}
+                  {...register('phoneNumber')}
+                  onChange={handlePhoneNumberChange}
+                  sx={{ width: '100%' }}
+                  error={errors['phoneNumber'] != undefined}
+                  helperText={errors['phoneNumber']?.message?.toString()}
```

Comment:
czemu tu jest value i onChange skoro to pole ma `register`? On już się powinien tym zająć

Reply:
Hmm, z tego co pamiętam coś nie działało z samym register

Reply:
Type '{ sx: { width: string; }; error: boolean; helperText: string | undefined; onChange: ChangeHandler; onBlur: ChangeHandler; ref: RefCallBack; name: "phoneNumber"; ... 8 more ...; continents: "EU"[]; }' is not assignable to type 'Omit<MuiTelInputProps, "ref">'.
  Types of property 'onChange' are incompatible.
    Type 'ChangeHandler' is not assignable to type '(value: string, info: MuiTelInputInfo) => void'.
      Types of parameters 'event' and 'value' are incompatible.
        Type 'string' is not assignable to type '{ target: any; type?: any; }'.

Taki pokazuje błąd jak usunę value i onChanged